support-frontend/assets/components/svgs/linkedin.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+
+// A monochrome version of the Twitter logo.
```

Comment:
```suggestion
// A monochrome version of the LinkedIn logo.
```
(or let's remove the line)